beacon_monitoring_platform_interface/lib/src/model/background_callback_result.dart

```
@@ -10,7 +10,7 @@ class BackgroundCallbackResult {
   static const _monitoringCallbackId = 'monitoringCallbackId';
   static const _monitoringResult = 'monitoringResult';
 
-  final int monitoringCallbackId;
+  final int? monitoringCallbackId;
```

I think monitoringCallbackId cannot be null, if we cannot assign callbackId in the native plugin side then we should throw an error